components/ArticleSection/ArticleSection.tsx

```diff
@@ -15,8 +16,8 @@ export const ArticleSection = memo<ArticleSectionProps>(({ article }) => {
   return (
     <section className={styles.section}>
       <h2 className={styles.heading}>
-        <Link href={article.blog.href}>
-          <a>
+        <Link href={addTrackingToLink(article.blog.href, { utm_medium: 'article_page' })}>
```

Comment:
Myślę, że warto również dodać tracking dla przejścia do samego artykułu, nie tylko bloga na którym się znajduje

Reply:
Jasne, ominąłem to.